src/main/java/hu/dungeonhunter/model/CharacterTypes.java

```diff
@@ -0,0 +1,15 @@
+package hu.dungeonhunter.model;
+
+import lombok.Getter;
```

Comment:
You can remove this import, it is not used.

Reply:
Thank you! Removed!